contentful/apps
# [INTEG-1295] - add getting started section for AIIG config page

## Purpose



This adds a getting started section, as well as a clear CTA for creating a new asset for users from the config page

## Approach

Config page aligns with generated comps for getting started section. Aside form comps, the decision to add a CTA for users to create and open a new asset has been added to demystify asset creation location

## New Look

<img width="276" alt="Screenshot 2023-09-21 at 7 14 59 AM" src="https://github.com/contentful/apps/assets/22120469/b0dacfc8-d0af-489e-aef0-cab6609e6679">

## Testing steps

- Install AI Image Generator to a space
- Instructions for getting started should match comps [here](https://www.figma.com/file/MyzNyHqDo1vjyAVGCMD852/AI-Image-Generator?type=design&node-id=64-40894&mode=design&t=mbiCAS64Tyf9C2Xr-0)
- `Try AI content generator` button should not be visible if app not installed
- `Try AI content generator` button should be visible when app is installed
